nodes/src/nodes/tool_mcp_client/mcp_sse_client.py

```
@@ -89,7 +89,7 @@ def __init__(
 
         self._reader_thread: threading.Thread | None = None
         self._stop = threading.Event()
-        self._incoming: "queue.Queue[dict]" = queue.Queue()
+        self._incoming: 'queue.Queue[dict]' = queue.Queue()
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)